### PR TITLE
Make WarpAffine test deterministic by removing random crop position from input image.

### PR DESCRIPTION
**Pull request by mzient**
Fixes randomly failing L0 test.

Signed-off-by: Michal Zientkiewicz <michalz@nvidia.com>

### CODE REVIEW

**Comment by mzient** [comment]
!build

File: dali/test/python/test_pipeline.py [hunk 1]
```diff
@@ -598,15 +598,14 @@ def __init__(self, batch_size, num_threads, device_id):
...
-            self.uniform = ops.Uniform(range = (0.0,1.0))
             self.iter = 0
 
         def define_graph(self):
             self.jpegs, self.labels = self.input()
             images = self.decode(self.jpegs)
             outputs = self.cmnp([images, images],
-                                crop_pos_x = self.uniform(),
-                                crop_pos_y = self.uniform())
+                                crop_pos_x = 0.5,
```
Comment (JanuszL) [inline dali/test/python/test_pipeline.py]: I think we can remove and use the default.

Comment (JanuszL) [reply dali/test/python/test_pipeline.py]: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html#cropmirrornormalize

Comment (mzient) [reply dali/test/python/test_pipeline.py]: Done.

**Pull request merged**